### ISSUE TITLE
Begäran låter lite bryskt

### ISSUE THREAD

**Issue by gabrielf** [bjorne/git-pa-svenska#11]
Jag föreslår förfrågan istället.

**Comment by davidhesselbom** [comment]
Anhållan?

**Comment by bkcsoft** [comment]
@davidhesselbom för att vänliga ord är ålderdomliga :trollface:

**Comment by magu** [comment]
Här behövs uppenbarligen lite militär nomenklatur: *v* hemställa, *s* hemställan

**Comment by davidhesselbom** [comment]
> hemställande, hemställning

Också på tok för gammalmodigt. Är det svenska ordförrådet verkligen såhär begränsat? :/

**Comment by magu** [comment]
> Är det svenska ordförrådet verkligen såhär begränsat?

Ja. Gammalmodigt är dock det nya nya!

**Comment by jdagerot** [comment]
Ja, jag förstår inte grejen med "för gammalmodigt" vi har ett fint och brett ordförråd på svenska, men om man kallar allt för "gammalmodigt" så blir det såklart begränsat.

**Comment by JonasOlson** [comment]
> _v_ hemställa; _s_ hemställande, hemställning

Fint ord, men substantivet tänker jag mig blir _hemställan_.

**Comment by emlun** [comment]
Vill man vara mindre humoristisk kan "förslag" vara ett alternativ. Tvåstavigt och bra!

>Vad tycker du om mitt ryckförslag?

**Comment by AndersBillLinden** [comment]
Ryckfråga? 😄
